homeassistant/components/canary/manifest.json

```diff
@@ -4,5 +4,6 @@
   "documentation": "https://www.home-assistant.io/integrations/canary",
   "requirements": ["py-canary==0.5.0"],
   "dependencies": ["ffmpeg"],
-  "codeowners": []
+  "codeowners": [],
```

Comment:
Please consider adding yourself as a code owner.

Reply:
I dont have a device to test against this  so i dont really want to be incharge of the support. this was just a aide project to modernize this integration for the community benefit

Reply:
I love that you are trying to make a contribution. This is a large PR and I'm concerned that not having a device to test with puts the integration at risk of breaking. Maybe we can find community testers. This is probably something one of the home assistant maintainers should weigh in on.